mobilecoind/strategies/accounts.py

```diff
@@ -12,6 +14,9 @@
 from random import randint
 from google.protobuf.empty_pb2 import Empty
 
+logging.basicConfig(stream = sys.stdout, level = logging.INFO)
```

Comment:
Is stdout where we want, or stderr or somewhere else too?

In general we should have a python logging module which sets config for all files and log with that instead of printing across all python files.

For now:
```suggestion
logging.basicConfig(stream = sys.stdout, level = logging.INFO, format='"{levelname} {module}:{lineno}: {message}", style="{"')
```

Reply:
We collect both stdout/stderr streams on the container, but in general all logging output should go to stdout by default.

I don't disagree that we could have a logging module, but that's way beyond the simple cleaning up the output I want to achieve here.

Reply:
Also the main output here is being run from a `kubectl exec` command via CI, so stdout is where we want the info so the CI system can display it.